flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -2222,6 +2222,9 @@ genOMP(Fortran::lower::AbstractConverter &converter,
     const bool genNested = !nextDir;
     const bool outerCombined = outermostLeafConstruct && nextDir.has_value();
     switch (leafDir) {
+    case llvm::omp::Directive::OMPD_masked:
+      TODO(currentLocation,
```

Comment:
Nit: Add comment with the spec (v4.5, if I'm not wrong, you can check the others below to make sure) section number for "MASKED".

Reply:
`masked` is a 5.1 feature

Reply:
Oh, if you want the Section number: it's Section 10.5 in OpenMP version 5.2

Reply:
I see that in a recent [commit](https://github.com/llvm/llvm-project/pull/90098/files#diff-496a295679ae3c43f8651c944a1bd9dca177ad2b5e4d7121f96938024e292bc1L2217) all section numbers are removed. The section numbers keeps changing unless some one needs to go to the exact spec. So I am more in favor of not adding it. Please let me know if you feel otherwise

Reply:
I won't argue for having the section numbers, as they are a moving target (like you rightfully say) and finding the constructs in the OpenMP spec is fairly easy. :-)